easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'libutempter'
+version = '1.1.6.2'
+
+homepage = 'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git'
+description = """libutempter is library that provides an interface for terminal
+ emulators such as screen and xterm to record user sessions to utmp and wtmp files."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+# Download link: http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=1.1.6-alt2;sf=tgz
+# source_urls = [
+#     'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h='; +
+#     version[0:5] + '-alt' + version[6:7] + ';sf=tgz']
```

Comment:
@SimonPinches I think we'll need to go with manual download for this one, so just including a comment like:

```python
# download via http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=1.1.6-alt2;sf=tgz
# and rename libutempter-1.1.6-alt2-3c6e555.tar.gz to libutempter-1.1.6.2.tar.gz
sources = [{
   'filename': SOURCE_TAR_GZ,
    'extract_cmd': "tar xfvz %s --strip 1",
}]

Reply:
@boegel Done (with an extra space before `'filename'`).